README.md

```diff
@@ -246,13 +247,115 @@ Example:
 conjur.variables().retrieveSecret("app/test-var");
 ```
 
+## Publication to Maven
```

Comment:
@h-artzi, @jvanderhoof: Shouldn't this go into a `CONTRIBUTING.md` file and not in the `README.md` file?  These are steps that the repo maintainers should take when they release a new version, right?  To use an analogy, this is like having a repo for a Rubygem and asking the gem user to publish it for their use.

Also, this information is great, but have we tested it?  Frankly, it seems pretty generic, as opposed to tailored to our org and this project.

Reply:
@alexkalish I can move this into a CONTRIBUTING.md file. 
@jvanderhoof according to this https://maven.apache.org/repository/guide-central-repository-upload.html we different options regarding where we can host these projects. I followed the instructions for Apache projects, hindsight it might not be the correct choice. The CONTRIBUTING.md will be changed accordingly. 

Currently approved repository hosting locations:
- Apache Software Foundation (for all Apache projects) 
- FuseSource Forge (focused on FUSE related projects)
- Nuiton.org
Other projects:
-  Open Source Software Repository Hosting (OSSRH)

It looks like the OSSRH (https://central.sonatype.org/pages/ossrh-guide.html) should fit our needs but let me know if you have any input regarding this change. If we choose to go down this path I might need input regarding how you want certain aspects named etc..

Reply:
Thanks, @h-artzi.  I might suggest moving this work into a separate branch, so that you aren't bundling with @oburstein-hub's work.

Reply:
irrelevant to me